etl/steps/__init__.py

```diff
@@ -73,22 +72,13 @@ def compile_steps(
 def to_dependency_order(
     dag: DAG, includes: List[str], excludes: List[str]
 ) -> List[str]:
-    # reverse the graph so that dependencies point to their dependents
-    graph = reverse_graph(dag)
```

Comment:
We had some methods expecting an input A->B to mean "A depends on B", while for others A->B meant "B depends on A". I think it's clearer now - I've flipped things around in a few places so that we always use the same directionality as the original `dag` passed in (namely, A->B means "A depends on B")